src/MediatR.Extensions.FluentValidation.AspNetCore/MediatR.Extensions.FluentValidation.AspNetCore.csproj

```diff
@@ -21,8 +21,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FluentValidation" Version="8.2.2" />
-    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="8.2.2" />
+    <PackageReference Include="FluentValidation" Version="9.0.1" />
+    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="9.0.1" />
```

Comment:
Do we really need newer version? It entail avaiability limitation for others

Reply:
There are breaking changes, `ValidationContext` - now generic.

Reply:
Sorry for my delay, vacation time. 
Generic validation context is also available in FluenValidation 8.0.0, so there is no need to upgrade version.
See: https://github.com/FluentValidation/FluentValidation/blob/8.0.0/src/FluentValidation/ValidationContext.cs

Reply:
I downgraded packages versions.